### ISSUE TITLE
Running the contributed command: 'extension.searchCTags' failed.

### ISSUE THREAD

**Issue by John15321** [rust-analyzer/rust-analyzer#9690]
Im just trying to use Ctrl+T in order to search for symbols like function names etc. But I get  this error:
```
Running the contributed command: 'extension.searchCTags' failed.
```

![image](https://user-images.githubusercontent.com/56229312/126884178-48606d25-ca1a-4c1b-8878-3989328b6ea6.png)

Im a beginner so I don't know much please help

**Comment by bjorn3** [comment]
What extensions do you have installed? `extension.searchCTags` is not a command registered by rust-analyzer. It doesn't know anything about CTags.

**Comment by John15321** [comment]
> What extensions do you have installed? `extension.searchCTags` is not a command registered by rust-analyzer. It doesn't know anything about CTags.

Okay, sorry it looks like the error was caused by this extension: https://github.com/jaydenlin/ctags-support
It overshadowed rust analyzers symbol searching. I uninstalled it and everything works now! I didn't think this could have been caused by another extension. Thanks!
![image](https://user-images.githubusercontent.com/56229312/126897152-3ae1e034-7ade-4d08-8ea6-d2808a5ebd28.png)
